src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1596,10 +1596,11 @@ namespace NServiceBus.Pipeline.Contexts
         protected LogicalMessageProcessingStageBehavior() { }
         public class Context : NServiceBus.Pipeline.Contexts.LogicalMessagesProcessingStageBehavior.Context
         {
-            public Context(NServiceBus.Unicast.Messages.LogicalMessage logicalMessage, System.Collections.Generic.Dictionary<string, string> headers, NServiceBus.Pipeline.Contexts.LogicalMessagesProcessingStageBehavior.Context parentContext) { }
+            public Context(NServiceBus.Unicast.Messages.LogicalMessage logicalMessage, System.Collections.Generic.Dictionary<string, string> headers, System.Type messageType, NServiceBus.Pipeline.Contexts.LogicalMessagesProcessingStageBehavior.Context parentContext) { }
```

Comment:
>I'm not sure I get it right but why is the constructor still needing LogicalMessage?

I only removed it from the outgoing pipeline. I have plans to remove it but I will do that in a separate pull

>There is no obsolete here because this has been introduce in V6 develop right?
Correct